WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController+FancyAlerts.swift

```diff
@@ -153,6 +155,30 @@ extension BlogDetailsViewController {
         }
     }
 
+    private func showCreateButtonAnnouncementAlert() {
+        guard noPresentedViewControllers else {
+            return
+        }
+
+        guard !UserDefaults.standard.createButtonAlertWasDisplayed else {
+            return
+        }
+
+        UserDefaults.standard.createButtonAlertWasDisplayed = true
+
+        let alert = FancyAlertViewController.makeCreateButtonAnnouncementAlertController { [weak self] (controller) in
+            controller.dismiss(animated: true)
+            if let url = URL(string: "https://wordpress.com/blog/") {
```

Comment:
I wonder if it would be feasible, once we remove the placeholder and use the actual url, to retrieve it from the backend in some way instead of hardcoding it

Reply:
Yeah, I don't know what the options are here. A remote config store like Firebase could help with this and turning on/off the announcement.